### PR TITLE
Adding new folio config files

### PR DESCRIPTION
**Pull request by tintumathew10**
# Description

Adding new folio config files.

### CODE REVIEW

File: conf/baremetal/folio1_1admin_4node_4client.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+globals:
+  -
+    ceph-cluster:
+      name: ceph
+      networks:
+        public: ['10.0.0.0/12']
+      nodes:
+        -
```
Comment (vamahaja) [inline conf/baremetal/folio1_1admin_4node_4client.yaml:8]: Correct indentation

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/reef/baremetal/nfs_folio3_3node_4client.yaml [hunk 2]
```diff
@@ -0,0 +1,153 @@
+---
+tests:
+  # -
```
Comment (vamahaja) [inline suites/reef/baremetal/nfs_folio3_3node_4client.yaml:3]: Remove commented tests

File: conf/baremetal/nfs_folio3_1admin_3node_4client.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
+globals:
+  -
```
Comment (vamahaja) [inline conf/baremetal/nfs_folio3_1admin_3node_4client.yaml:2]: Fix indentation

File: conf/baremetal/folio2_1admin_4node_4client.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
+globals:
+  -
```
Comment (vamahaja) [inline conf/baremetal/folio2_1admin_4node_4client.yaml:2]: Fix indentation

File: conf/baremetal/folio1_1admin_4node_4client.yaml [hunk 5]
```diff
@@ -0,0 +1,81 @@
+globals:
+  -
```
Comment (vamahaja) [inline conf/baremetal/folio1_1admin_4node_4client.yaml:2]: Fix indentation

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: conf/baremetal/folio1_1admin_4node_4client.yaml (hunk 5 above)
Comment (tintumathew10) [reply conf/baremetal/folio1_1admin_4node_4client.yaml:2]: done

**Review by vamahaja — APPROVED** [review]

**Pull request merged**